### ISSUE TITLE
[Mentor] mentor notes, 2nd slide bar

### ISSUE THREAD

**Issue by joshyellin** [greenido/lpa-1#95]
Didn't we want to remove the question "Were the attendees open and receptive?"

I don't think it's all that useful... it's usually reflected in the "effective" question and the notes.

wdyt?

**Comment by greenido** [comment]
Yep... Let's think if there is another 1-5 question we wish here or not.

**Comment by joshyellin** [comment]
I have always wanted a question of "Should we schedule a follow-up meeting between you and this startup?" "If so, in how many days?"

If you agree, we can think of a good way to ask this. But from my experience, this would have been extremely helpful, and we would ideally ask both the mentors and the startups.

wdyt?

Maybe it's like this... "Should we schedule a follow-up meeting for you and this startup? (checkbox)
And then.. "In how many days?" (small field)

Without this info from the mentor, trying to know how much homework they gave, if a second meeting during bootcamp is necessary, maybe not important for a 30-60 days... it all depends. Right now we don't have that info anywhere.

**Comment by greenido** [comment]
Let's think about it.
It's important info to gain from the mentors - but might be better in other channels (e.g. slack).

**Comment by joshyellin** [comment]
Sg, let's def talk about best way to capture it. Let's also have a chat about ways we can improve the matchmaking process. Theres A LOT of room for improvement on this front based on previous bootcamps, so def worth a good talking over together.
